src/plugins/laya-la-scmc/view.vue

```diff
@@ -249,7 +249,7 @@ export default {
      */
```

Comment:
Please update the "Last Updated" Line in function comment (also for `diffSolution`)